src/js/angular/repositories/controllers.js

```diff
@@ -270,6 +271,16 @@ function LocationsAndRepositoriesCtrl($scope, $modal, toastr, $repositories, Mod
     };
 
     $repositories.init();
+
+    function removeCachedGraphsOnDelete(repoId) {
+        angular.forEach(LocalStorageAdapter.keys(), function (key) {
+            // remove everything but the hide prefixes setting, it should always persist
+            if (key.startsWith("classHierarchy-selectedGraph-" + repoId) || key.startsWith("dependencies-selectedGraph-" + repoId)) {
```

Comment:
These `"classHierarchy-selectedGraph-" + repoId` and `"dependencies-selectedGraph-" + repoId` can be computed once outside of the loop.